src/main/java/net/minecraftforge/fml/common/ILanguageAdapter.java

```diff
@@ -33,6 +34,11 @@
     public void setProxy(Field target, Class<?> proxyTarget, Object proxy) throws IllegalArgumentException, IllegalAccessException, NoSuchFieldException, SecurityException;
     public void setInternalProxies(ModContainer mod, Side side, ClassLoader loader);
 
+    default public IStaticContainer getStaticContainer(Class<?> clazz, ClassLoader classLoader) throws ClassNotFoundException, NoSuchFieldException, IllegalAccessException
+    {
+        return new IStaticContainer.ClassStaticContainer(clazz);
+    }
+
     public static class ScalaAdapter implements ILanguageAdapter {
```

Comment:
I deliberately didn't change the Scala adapter, since this basically goes in conjunction with #4029. Thanks for reminding me to mention that PR, though :D